src/QMCHamiltonians/SOECPComponent.h

```diff
@@ -116,6 +118,7 @@ class SOECPComponent : public QMCTraits
 
   RealType calculateProjector(RealType r, const PosType& dr, RealType sold);
 
+  RealType evaluateOneFast(ParticleSet& W, const int iat, const TrialWaveFunction& psi, const int iel, const RealType r, const PosType& dr);
```

Comment:
Could you please rename it? "Fast" is not really helpful. Please also add documentation with a brief description of the algorithm.

Reply:
done